pkg/lib/registry/registry.go

```diff
@@ -39,6 +39,21 @@ func (r RegistryUpdater) AddToRegistry(request AddToRegistryRequest) error {
 		return err
 	}
 
+	// use multi-image loader for add request with multiple image bundles
+	if len(request.Bundles) > 1 {
```

Comment:
should we just always use the multi image loader? why do we need a separate loader?

Reply:
I agree, this looks kind of funny. I wanted to preserve the old code path, just in case, for now. But conceptually yeah I guess we should handle all `index add` calls through the new codepath